packages/terra-consumer-nav/src/components/nav-help/NavHelp.jsx

```diff
@@ -32,14 +32,15 @@ const propTypes = {
   /**
    * A unique id set to the help button that will be referred in help menu/popup .
    */
-  id: PropTypes.string.isRequired,
+  id: PropTypes.string,
   /**
    * Injected react-intl formatting api
    */
   intl: intlShape.isRequired,
 };
 
 const defaultProps = {
+  id: '--terra-consumer--nav-help-button',
```

Comment:
any reason using css custom property format here?

Reply:
figured it'd be unique enough to not cause conflicts anywhere. Can change if there is a standard to follow here.

Reply:
that format is reserved for css custom variables, so i would remove the double dash